ember-autofocus-modifier/src/modifiers/autofocus.ts

```diff
@@ -0,0 +1,82 @@
+import { modifier } from "ember-modifier";
+import { next } from "@ember/runloop";
+
+const focusableElements = [
+  "BUTTON",
+  "SUMMARY",
+  "IFRAME",
+  "INPUT",
+  "SELECT",
+  "TEXTAREA",
+] as const;
+
+const DEFAULT_SELECTOR =
+  "input:not([disabled]):not([readonly]),textarea:not([disabled]):not([readonly])";
+
+interface ModifierArgs {
+  Element: HTMLElement;
+  Args: {
+    Positional: [string?] | undefined; // Optional selector
+    Named: {
+      disabled?: boolean; // Optional 'disabled' parameter
```

Comment:
Are these comments necessary? It seems clear to me, especially if we take the time to look at the modifier implementation

Reply:
I think the first one in particular is needed because by reading the type alone I'm a bit confused to be honest. That function signature is a bit weird 😢 
I thought about removing the second one, but in the end I decided that it's better to keep both for consistency.

Reply:
Got it, sounds good 👍